src/app/src/components/UserPost.tsx

```diff
@@ -193,34 +196,40 @@ const UserPost: React.FC<postItem> = (props?) => {
   };
 
   //Navigate to user's profile from userpost
-  const HandleNavigation = () => { 
-    navigate(`/profile/${props?.postAuthor?.id.split('/').pop()}`) 
+  const HandleNavigation = () => {
+    navigate(`/profile/${props?.postAuthor?.id.split("/").pop()}`);
+  };
+
+  const navigateToComments = () => {
+    navigate(
+      `/authors/${props?.postAuthor?.id.split("/").pop()}/posts/${props?.post?.id}/comments`,
```

Comment:
Same issue here.
This looks like the backend route. We wanted the frontend route to just be `path="/post/:postId"` where you can see the post and all it's comments.

Reply:
@Sutanshu Same here again, should be of the form /profile/${props?.postAuthor?.id.split("/").pop()}/post/${props?.post?.id}` same as App.tsx